src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const fixedRobot = {};
+
+  for (const key in robot) {
+    if (Object.hasOwnProperty.call(fixedRobot, robot[key])) {
```

Comment:
Use 'fixedRobot.hasOwnProperty' instead of 'Object.hasOwnProperty.call' for better readability

Reply:
Sure. However, isn't it better to prevent a scenario when a built-in method is overridden?